verifier/src/lib.rs

```diff
@@ -60,7 +60,29 @@ const SUCCESS_MESSAGE_INDICATOR: &str = "- [x]";
 /// Failure checkbox indicator
 const FAILURE_MESSAGE_INDICATOR: &str = "- [ ]";
 
-pub(crate) fn choice_to_status_message(choice: Choice) -> &'static str {
+/// Provides a success/fail indicator to [`VerificationMessage`] implementations
+///
+/// ```
+/// use core::fmt::{Display, Formatter};
+/// use mc_attestation_verifier::choice_to_status_message;
+/// use subtle::Choice;
+///
+/// pub struct Foo(Choice);
+///
+/// impl Display for Foo {
+///     fn fmt(&self, f: &mut Formatter<'_>) -> core::fmt::Result {
+///         let status = choice_to_status_message(self.0);
+///         write!(f, "{status} some message here")
+///     }
+/// }
+///
+/// let fail = Foo(0.into());
+/// assert_eq!(fail.to_string(), "- [ ] some message here");
+///
+/// let success = Foo(1.into());
+/// assert_eq!(success.to_string(), "- [x] some message here");
+/// ```
+pub fn choice_to_status_message(choice: Choice) -> &'static str {
```

Comment:
Thoughts on if this should be a Trait on `Choice`? and perhaps `status_indicator`

I'm also fine leaving as is.

Reply:
Either way is fine.

Reply:
leaving for now. 
I need to better understand when to use traits. I think they make sense when more than one thing will need to implement the behavior, but for single extensions, functions may be better, not sure yet.